### DISCUSSION TITLE
Windows paths don't support the pipe `|` operator

### DISCUSSION THREAD

**Discussion by dhirschfeld** [copier-org/copier#1736] [Q&A] [answered]
It seems Windows paths don't support the pipe operator:

![image](https://github.com/user-attachments/assets/4e04298f-9aec-47f4-9eda-b80fbe10c731)

I'm wanting to normalize the `project_name` input (`s/-/_/`) so the folder name represents an importable Python package.

This works great on Linux. Does anyone have any idea how I could do the same on Windows? :pray:

**Comment by sisp** [comment, marked as answer]
Correct, the pipe operator doesn't work in Windows paths. I suggest to use a [computed value](https://copier.readthedocs.io/en/stable/faq/#how-to-create-computed-values) in this case which makes the Jinja expression more readable as well.
